Gemfile

```diff
@@ -15,7 +15,9 @@ gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'capybara'
+  gem 'launchy'
   gem 'rspec-rails'
+  gem 'shoulda-matchers'
```

Comment:
not actually sure i needed these yet, but was struggling getting rspec to `get '/api/v1/trails?location=denver,co'`
i was getting 'i don't know what get is' until i added `type: :request` to the describe line. not sure i've experienced that before, but i haven't really rspec tested api endpoints. they've always been view endpoints.